Extension/src/LanguageServer/client.ts

```diff
@@ -1856,9 +1856,11 @@ export class DefaultClient implements Client {
         if (!currentProvider.isReady) {
             return;
         }
+
         await Promise.all([
```

Comment:
Does `clearCustomConfigurations` need to complete before `handleRemoveAllCodeAnalysisProblems` ?

if not, then I'm ok with this.

if it does, just await them individually

Reply:
The order doesn't matter between those two calls.